src/annotation/index.ts

```diff
@@ -1,4 +1,4 @@
-import { Toolbox, Toolboxes } from "@/Toolboxes";
+import { Toolbox, Toolboxes } from "../Toolboxes";
```

Comment:
Why this change?

Reply:
I changed that path, because Jest didn't find the module when using the alias and I didn't want to venture into config territory.